.idea/misc.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same question for this file

Reply:
I think can be kept in git. Not user specific and will change very rarely.